src/fal/planner/tasks.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import annotations
+
+import argparse
+import sys
+import traceback
+import uuid
+from contextlib import contextmanager
+from dataclasses import dataclass, field
+from functools import cached_property
+from typing import Iterator, Any
+
+from dbt.logger import GLOBAL_LOGGER as logger
+
+from fal.node_graph import FalScript
+from fal.utils import print_run_info
+from faldbt.project import DbtModel, FalDbt, NodeStatus
+
+SUCCESS = 0
+FAILURE = 1
```

Comment:
should this be an enum?

Reply:
I don't think it works for now (it would be basically `SUCCESS`, since anything beside it is technically an error). Maybe in the future when we implement the `SKIPPED` though.